core/src/test/scala/unit/kafka/admin/DeleteConsumerGroupsTest.scala

```diff
@@ -88,10 +88,12 @@ class DeleteConsumerGroupsTest extends ConsumerGroupCommandTest {
     val service = getConsumerGroupService(cgcArgs)
 
     TestUtils.waitUntilTrue(() => {
-      service.listGroups().contains(group)
+      service.collectGroupMembers(false)._2.get.size == 1
```

Comment:
Thanks for improving this check. Do you think it's worth doing the same in a few other unit tests in this file?

Reply:
I went over the other tests, and none of them required a non-empty group specifically. Did I miss something?

Reply:
You're right, sorry my mistake.